lib/loader.js

```diff
@@ -33,6 +33,7 @@ module.exports = function svgReactLoader (source) {
     var raw            = params.raw;
     var xmlnsTest      = params.xmlnsTest;
     var classIdPrefix  = params.classIdPrefix || false;
+    var uniqueIdPrefix = params.uniqueIdPrefix || false;
```

Comment:
Would it make sense to use the value, after interpolation, of `classIdPrefix` as a default?

Reply:
(interpolation will be fixed as soon as #107 is merged, btw)

Reply:
I'm not sure if that would work as intended.

Using the same prefix would across all instances and files would collide too, no?

It might be more relevant to allow tokenizing within a default string, a la: `[name]-[hash]-`